tools/rna_tools/sortmerna/tool_dependencies.xml

```diff
@@ -2,17 +2,46 @@
 <tool_dependency>
     <package name="sortmerna" version="2.0">
         <install version="1.0">
-            <actions>
-                <action type="download_by_url" target_filename="sortmerna-2.0.tar.gz">https://github.com/biocore/sortmerna/archive/2.0.tar.gz</action>
-                <action type="autoconf"/>
-                <action type="set_environment">
-                    <environment_variable name="SORTMERNADIR" action="set_to">$INSTALL_DIR</environment_variable>
-                    <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR/bin</environment_variable>
-                </action>
-            </actions>
+            <actions_group>
+                <actions architecture="x86_64" os="linux">
+                        <action type="download_by_url" target_filename="sortmerna-2.0.tar.gz">https://github.com/biocore/sortmerna/archive/2.0.tar.gz</action>
+                        <action type="autoconf"/>
+                        <action type="change_directory">../</action>
+                        <action type="make_directory">$INSTALL_DIR/sortmerna-2.0/rRNA_databases/</action>
```

Comment:
I'd use just `$INSTALL_DIR/rRNA_databases`, but that's just a personal preference.